.github/workflows/run_tests.yml

```
@@ -69,6 +69,7 @@ jobs:
     name: Test "${{ inputs.marker }}"
     runs-on: ubuntu-latest
     env:
+      ASSEMBLYAI_API_KEY: ${{ secrets.ASSEMBLYAI_API_KEY }}
```

actionlint (1.7.11)

[error] property "assemblyai_api_key" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; github_token: string}

(expression)